iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -137,10 +137,6 @@ class ThreadViewController: DiffableCollectionViewController<MessageSequenceSect
         if self.collectionView.isTracking {
             self.detailView.alpha = 0.0
         }
-    }
-
-    override func viewDidDisappear(_ animated: Bool) {
-        super.viewDidDisappear(animated)
```

Comment:
The keyboard manager observers were getting added multiple times for the thread vc. I now make sure to clean them up on viewWillDisappear, not viewDidDisappear.